tests/sles4sap/saptune/mr_test.pm

```diff
@@ -276,8 +297,12 @@ sub run {
         # be tested with an override file
         $self->test_note($test) if ($test ne "1805750");
         $self->test_override($test);
-    } elsif ($test =~ m/^(x86_64|ppc64le)$/) {
+    } elsif ($test eq "x86_64") {
         $self->test_x86_64 if (check_var('BACKEND', 'ipmi'));
+        $self->test_bsc1152598;
+    } elsif ($test eq "ppc64le") {
+        $self->test_ppc64le;
+        $self->test_bsc1152598;
```

Comment:
I found this `if [...] elsif` condition more and more difficult to read :D

Please consider refactoring this for a future PR if it's possible. I don't want to block BV for such a small thing ;-)

Reply:
Perl (but Python too) lacks a C-like `switch` statement.